exaudfclient/base/exaudfclient.cc

```diff
@@ -13,7 +13,7 @@
 #include <dlfcn.h>
```

Comment:
Add comment, that this lib is for the dynamic loading which is not required for udf plugins

Reply:
Done